### DISCUSSION TITLE
“System.OutOfMemoryException”类型的未经处理的异常在 nanoFramework.Graphics.dll 中发生       ++++ Exception System.OutOfMemoryException - CLR_E_OUT_OF_MEMORY (3) ++++     ++++ Message:      ++++ nanoFramework.UI.Bitmap::.ctor [IP: 0000] ++++     ++++ nanoFramework.UI.DisplayControl::get_FullScreen [IP: 0015] ++++  Uncaught exception

### DISCUSSION THREAD

**Discussion by zhangqiang199226** [nanoframework/Home#1269] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/74021799/231195657-4674d3dc-b8fa-42af-b3b7-bc8e1d8de09c.png)
![image](https://user-images.githubusercontent.com/74021799/231196025-5c6eec8b-f948-4d08-a6d8-bd840d0cfccf.png)
![image](https://user-images.githubusercontent.com/74021799/231196193-11430ca4-e235-4ea0-b1c0-71d502e2d6b2.png)

**Comment by Ellerbach** [comment]
It's because your board does not seems to have PSRAM. So it does not have enough memory for a full frame buffer. Try with a smaller allocation.

**Reply by zhangqiang199226** [reply]
This ESP32 module has a built-in 64-Mbit PSRAM, providing flexible additional storage space and data processing capabilities.

**Reply by zhangqiang199226** [reply]
![image](https://user-images.githubusercontent.com/74021799/231205102-7504a8c5-c50e-4c2e-88b8-f6fdc6f75783.png)

**Reply by Ellerbach** [reply]
ok, so then, can you try to pass a very large buffer for the memory setup?

**Reply by zhangqiang199226** [reply, marked as answer]
![image](https://user-images.githubusercontent.com/74021799/231680818-0953d76d-b850-47ea-9edc-09ccfb6f20d8.png)
Now the execution has reached the point of throwing an exception。
